media/css/firefox/fxos-nav.less

```diff
@@ -0,0 +1,576 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/lib.less";
+
+// mixin for sprite images
+.fxosnav-sprite(@top) {
+    background-image: url(/media/img/firefox/os/nav-sprite.png);
+    background-position: 0 @top;
+
+    @media (-webkit-min-device-pixel-ratio: 1.5),
+       (                min-resolution: 1.5dppx),
+       (                min-resolution: 144dpi) {
+
+        .background-size(98px 115px);
+        background-image: url(/media/img/firefox/os/nav-sprite-high-res.png);
+    }
+}
+
+// big left hand logo
+.fxosnav-logo {
+    .fxosnav-sprite(0);
+    .image-replaced();
+    .transform(translateY(30px));
+    .transition(-webkit-transform 0.3s ease);
+    .transition(transform 0.3s ease);
```

Comment:
On second thought, I agree with you that worrying about a transition for an old webkit browser isn't really worth changing up the code. The `transform` will still work. Will simply remove `.transition(-webkit-transform 0.3s ease);`